app/budgetpanel.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, OnInit } from '@angular/core';
+
+export interface Category {
+    id: number;
+    name: string;
+}
+
+@Component({
+  selector: 'budget-panel',
+  styles: [`
+    budgetpanel-container {
+        background-color: #cfc;
+        display: block;
+    }
+    
+    .container-fluid {
+        padding: 10px;
+        max-width: 1024px;
+    }
+  `],
+  template: `     
+        <div class="container-fluid">
+            <div class="row">  
+                <template ngFor let-category let-i="index" [ngForOf]="categories" [ngForTrackBy]="trackById">                                    
+                    <div class="col-sm-4">
+                        <category [category]="category"></category>  
+                    </div>                  
+                </template>
+            </div>
+        </div>    
+  `,
+})
+
+export class BudgetPanel implements OnInit {
+  categories: Array<Category>;
+  ngOnInit() {
+      this.categories = [{id: 1, name: 'category1'}, {id: 2, name: 'category2'}, {id: 3, name: 'category3'}, {id: 4, name: 'category4'}, {id: 5, name: 'category5'}, {id: 6, name: 'category6'}, {id: 7, name: 'category7'}];
```

Comment:
This is great!

The actual data can be found here:
http://next.obudget.org/search/get/document/budget-functional-aggregates

the big circles should be named after 'func_cls_title_1' and the small ones should be named after 'func_cls_title_2'.